From bda632a69ff44114f9faf689a5cd4af1fe578d66 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 15 Feb 2018 17:33:44 -0800
Subject: [PATCH] TRITON-139 Triton's config-agents should poll at 120s
 interval as a stopgap Reviewed by: Orlando Vazquez <orlando@joyent.com>
 Approved by: Orlando Vazquez <orlando@joyent.com>

---
 lib/setup_config_agent.js | 4 ++--
 package.json              | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/setup_config_agent.js b/lib/setup_config_agent.js
index 2ba2187..f3bc21a 100644
--- a/lib/setup_config_agent.js
+++ b/lib/setup_config_agent.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2016 Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -111,7 +111,7 @@ main()
 
 		config = {
 			logLevel: 'info',
-			pollInterval: 60 * 1000,
+			pollInterval: 2 * 60 * 1000,
 			sapi: {
 				url: url
 			},
diff --git a/package.json b/package.json
index 5e5194c..7291ad3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-scripts",
   "description": "Shared boot scripts for SDC core service zones",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "license": "MPL-2.0"
-- 
2.21.0

